tests/test_filesystem.py

```diff
@@ -373,7 +368,7 @@ def test_resync_no_src_wpsoutputs(self):
 
 
 @pytest.mark.filesystem
-class TestFileSystemWpsOutputsUser(TestFileSystem):
+class TestFileSystemWpsOutputsUser(BaseTestFileSystem):
     """
     FileSystem tests specific to the user wps outputs data.
```

Comment:
Can you add a ``.. note::`` with a very brief summary and a reference to the doc where ``others`` permission is explained instead of using ``user``/``group``? Reading the tests without this context feels odd due to the `660`/`664` permissions. 

Also add a 
```rst
.. seealso::
    - :func:`cowbird.utils.update_filesystem_permissions` 
```

And maybe you need to adjust its description. It mentions "(user only)" although it is working on `others` permissions. 
The function itself could also reference the docs for the detailed explanation.